src/aks-preview/HISTORY.md

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.5.11
++++++
+* Add get OS options support
```

Comment:
hey @JunSun17 I have another change https://github.com/Azure/azure-cli-extensions/pull/3345 is targeting for 0.5.11, mind add my change in your history?

> * Fix wrong behavior when enabling pod identity addon for cluster with addon enabled

Reply:
No problem, I have add yours to this bump too.